### DISCUSSION TITLE
Enhance Outline view to remember your last collapsed state

### DISCUSSION THREAD

**Discussion by Gregory1701** [codefori/vscode-ibmi#1308] [IBM i] [answered]
Hey guys,
I have an idea for an enhancement to the outline view.  Most of the time, I tend to operate in a collapsed outline view.  Thus, every source member I open I have to go collapse the outline view.  IMO this seems like a good enhancement to have the extension just remember the collapsed state of the outline view, not for each source member, just for overall.  I suppose this could be a setting, but my thought is just remember which state it was at the last time you closed VS Code, then pick up where you left off the next time you open VS Code for i.

What do you guys think?

**Comment by worksofliam** [comment]
Good idea but out of our control. This kind of state is handled entirely be VS Code and not something we can do.

I would see if any settings existing for this, before raise an issue on the VS Code with your idea.

**Reply by Gregory1701** [reply, marked as answer]
Ah!  I found the setting.
![image](https://github.com/halcyon-tech/vscode-ibmi/assets/132368707/dbfa016a-3cc6-46b9-ac24-be407a1a69c7)
